generator/sig_readme.tmpl

```diff
@@ -13,18 +13,35 @@
 {{- end }}
 {{- end }}
 
-{{ end -}}
-## Leads
-{{- range .Leads }}
+{{- end }}
+{{- if .Leadership }}
+
+## Leadership
+{{- if .Leadership.Chairs }}
+
+### Chairs
+The Chairs of the SIG run operations and processes governing the SIG.
```

Comment:
maybe replace "processes governing" with just "govern"?

Reply:
I don't disagree, but wanted to use the similar wording to the short template the steering committee provided here: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance-template-short.md#roles

Reply:
I'd prefer keeping the @cblecker's wording.